facebook/delphiFacebook/R/contingency_tables.R

```diff
@@ -0,0 +1,1068 @@
+    ## free response
+    # Either number ("n"; can be averaged although may need processing) or text ("t")
+    "n_hh_num_sick" = "hh_number_sick", # A2
+    "n_hh_num_children" = "A5_1",
+    "n_hh_num_adults" = "A5_2",
+    "n_hh_num_seniors" = "A5_3",
+    # "t_zipcode" = "A3", # zip5
+    "n_cmnty_num_sick" = "A4",
+    "t_symptoms_other" = "B2_14_TEXT",
+    "t_unusual_symptoms_other" = "B2c_14_TEXT",
+    "t_gender_other" = "D1_4_TEXT",
+    "n_days_unusual_symptoms" = "B2b",
+    "n_contact_num_work" = "C10_1_1",
+    "n_contact_num_shopping" = "C10_2_1",
+    "n_contact_num_social" = "C10_3_1",
+    "n_contact_num_other" = "C10_4_1",
+    "n_hh_num_total" = "hh_number_total", # A2b from Waves <4 and summed A5 from Wave 4
+    "n_highest_temp_f" = "Q40",
+    "n_hh_num_children_old" = "D3", # Wave 1, etc versions of A5
+    "n_hh_num_adults_not_self" = "D4",
+    "n_hh_num_seniors_not_self" = "D5",
+    
+    
+    ## binary response (b)
+    # False (no) is mapped to 2 and True (yes/agreement) is mapped to 1
+    "b_consent" = "S1",
+    "b_hh_fever" = "hh_fever", # A1_1
+    "b_hh_sore_throat" = "hh_soar_throat", # A1_2
+    "b_hh_cough" = "hh_cough", # A1_3
+    "b_hh_shortness_of_breath" = "hh_short_breath", # A1_4
+    "b_hh_difficulty_breathing" = "hh_diff_breath", # A1_5
+    "b_tested_ever" = "B8",
+    "b_tested_14d" = "t_tested_14d", # B10; "No" coded as 3, but dealt with in conversion to "t_tested_14d"
+    "b_wanted_test_14d" = "t_wanted_test_14d", # B12
+    "b_state_travel" = "C6", # c_travel_state
+    "b_contact_tested_pos" = "C11",
+    "b_contact_tested_pos_hh" = "C12",
+    "b_hispanic" = "D6",
+    "b_worked_4w" = "D9",
+    "b_worked_outside_home_4w" = "D10",
+    "b_took_temp" = "B3",
+    "b_flu_shot_12m" = "C2",
+    "b_worked_outside_home_5d" = "c_work_outside_5d", # C3
+    "b_worked_healthcare_5d" = "C4",
+    "b_worked_nursing_home_5d" = "C5",
+    "b_anxious" = "mh_anxious", # Binary version of C8_1
+    "b_depressed" = "mh_depressed", # Binary version of C8_2
+    "b_isolated" = "mh_isolated", # Binary version of C8_3
+    "b_worried_family_ill" = "mh_worried_ill", # Binary version of C9   
+    "b_public_mask_often" = "c_mask_often", # Binary version of C14
+    "b_tested_pos_14d" = "t_tested_positive_14d", # B10a; binary with an "I don't know" (3) option
+    "b_tested_pos_ever" = "B11", # binary with an "I don't know" (3) option
+    "b_have_cli" = "is_cli", # Based on symptoms in A1
+    "b_have_ili" = "is_ili", # Based on symptoms in A1
+    "b_cmnty_have_cli" = "community_yes",
+    "b_hh_or_cmnty_have_cli" = "hh_community_yes",
+    
+    ## multiple choice (mc)
+    # Can only select one of n > 2 choices
+    "mc_state" = "A3b",
+    "mc_mask_often" = "C14",
+    "mc_anxiety" = "C8_1",
+    "mc_depression" = "C8_2",
+    "mc_isolation" = "C8_3",
+    "mc_worried_family_ill" = "C9",
+    "mc_financial_worry" = "C15",
+    "mc_gender" = "D1",
+    "mc_age" = "D2",
+    "mc_race" = "D7",
```

Comment:
D7 actually allows the respondent to select multiple races. However, @dshemetov has found that in most cases, multirace combinations are rare and would not meet the privacy threshold. How would you like to handle that here? They could be coded into a single "multirace" group, for example, instead of including all of the powerset of possible race combinations

Reply:
(driveby; ignore if nonsensical) could we treat D7 the way we do zip codes that cross county lines; ie distribute the response across all respondent races weighted by 1/length(D7)?

Reply:
> D7 actually allows the respondent to select multiple races

Thanks for the correction!

There's also the possibility of moving multirace responses to the "other" category and listing the response codes as the "other" text. I'll look into how frequently this type of response occurs – it's possible they're rare enough that even a "multirace" category would always be censored due to low sample size.

Reply:
You  can poke @chinandrew or @dshemetov about the counts -- they already  did some analysis of the frequency of different options while considering differential privacy.

Reply:
From old slacks between me/alex/dmitry, these were the results looking at a file in early Nov:
```
250k responses
~65k nans 
~180k responded with only 1 race
~5k responded with >1 race
```